### PR TITLE
triggers: remove fast

### PR DESCRIPTION
**Pull request by cloudformdesign**

### CODE REVIEW

**Comment by cloudformdesign** [comment]
crap, forgot to remove from `blocks` file

File: blocks.py [hunk 1]
```diff
@@ -25,13 +25,13 @@ class CounterSafeSimulator(
         SafeTrigger,
         Block):
 
-    version = VersionProperty('1.0.0')
+    version = VersionProperty('1.0.1')
 
 
 @Discoverable(DiscoverableType.block)
 class IdentityFastSimulator(
```
Comment (mattdodge) [inline blocks.py]: I think we can get rid of this simulator too, don't think it's in use

**Comment by mattdodge** [comment]
Please remove from README too

**Comment by cloudformdesign** [comment]
this looks like it will have to be fixed to merge. Let's merge the other branches and then remake this branch

**Pull request merged**